src/engine/renderer/tr_shade.cpp

```diff
@@ -2806,7 +3001,7 @@ Render tesselated data
 */
 void Tess_End()
 {
-	if ( ( tess.numIndexes == 0 || tess.numVertexes == 0 ) && tess.multiDrawPrimitives == 0 )
+	if ( ( tess.numIndexes == 0 || tess.numVertexes == 0 ) && tess.multiDrawPrimitives == 0 && !tr.drawingSky )
```

Comment:
Normally sky is drawn only when sky brushes are processed in `RB_RenderDrawSurfaces()`, which are only added in `R_AddWorldSurfaces()`. Since the latter is not being called anymore, the sky wouldn't be drawn without this check as there're no vertexes in Tess at this point.